utility/importer/chrome_importer.cc

```diff
@@ -147,15 +168,16 @@ void ChromeImporter::StartImport(const importer::SourceProfile& source_profile,
 }
 
 void ChromeImporter::ImportHistory() {
-  base::FilePath history_path =
-    source_path_.Append(
+  base::FilePath history_path = source_path_.Append(
       base::FilePath::StringType(FILE_PATH_LITERAL("History")));
   if (!base::PathExists(history_path))
     return;
 
+  CreateCopyFile copy_history_file(history_path);
   sql::Database db;
-  if (!db.Open(history_path))
+  if (!db.Open(copy_history_file.copied_file_path())) {
```

Comment:
re: https://github.com/brave/brave-core/pull/5336/files#r423416068 this will crash if either the temp file creation or the copy fails